app/models/admin_set.rb

```diff
@@ -74,6 +74,11 @@ def permission_template
     Hyrax::PermissionTemplate.find_by!(admin_set_id: id)
   end
 
+  def delete_permission_template_access
+    Hyrax::PermissionTemplateAccess.where(:permission_template_id => permission_template.id).delete_all
```

Comment:
Doesn't the PT have a `dependent: :destroy` relationship with the PTAs? I don't *think* this should be necessary but I could be mistaken.